src/arguments/shinywager.ts

```diff
@@ -1,30 +1,19 @@
 import { LanguageKeys } from '#lib/i18n/languageKeys';
 import { ApplyOptions } from '@sapphire/decorators';
-import type { Message } from 'discord.js';
-import { Argument, ArgumentOptions, Possible } from 'klasa';
+import { ExtendedArgument, ExtendedArgumentContext, ExtendedArgumentOptions } from '@sapphire/framework';
 
-@ApplyOptions<ArgumentOptions>({ aliases: ['wager'] })
-export default class ShinyWager extends Argument {
-	public async run(arg: string, possible: Possible, message: Message): Promise<number> {
-		if (!arg) throw await message.resolveKey(LanguageKeys.Resolvers.InvalidInt, { name: possible.name });
-
-		const number = Number(arg) as ArrayValues<typeof ShinyWager.kValidBetAmounts>;
-		if (!Number.isInteger(number)) throw await message.resolveKey(LanguageKeys.Resolvers.InvalidInt, { name: possible.name });
-		if (!ShinyWager.kValidBetAmounts.includes(number)) {
-			throw await message.resolveKey(LanguageKeys.Resolvers.InvalidWager, {
-				bet: number,
-				validAmounts: ShinyWager.kValidBetAmounts.map((a) => a.toString())
-			});
-		}
-
-		return this.integerArg.run(arg, possible, message);
+@ApplyOptions<ExtendedArgumentOptions<'integer'>>({ aliases: ['wager'], baseArgument: 'integer' })
+export default class ShinyWager extends ExtendedArgument<'integer', number> {
```

Comment:
Are you still wanting to `export default`? You've changed the other arguments thus far to `export`.

Reply:
Yeah, I'm not done replacing all of those yet.

Reply:
Fixed in 4b8b91a!
![image](https://user-images.githubusercontent.com/24852502/106893663-2b525e00-66ee-11eb-96fe-5dc3985f5ae0.png)